SECURITY.md

```diff
@@ -34,8 +27,34 @@ announcement, and may ask for additional information or guidance.
 > [!NOTE]  
 > You can find more information about our process in [this guide](https://github.com/expressjs/security-wg/blob/main/docs/handle_security_reports.md)
 
-Report security bugs in third-party modules to the person or team maintaining
-the module.
+
+### Reporting Security Bugs via GitHub Security Advisory (Preferred)  
+
+The preferred way to report security vulnerabilities is through 
+[GitHub Security Advisories](https://github.com/advisories). 
+This allows us to collaborate on a fix while maintaining the 
+confidentiality of the report.  
+
+To report a vulnerability
+([docs](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability)):  
+1. Visit the **Security** tab of the affected repository on GitHub.  
+2. Click **Report a vulnerability** and follow the provided steps.  
+
+This process applies to any repositories within the Express ecosystem. 
+If you are unsure whether a repository falls under this policy, 
+feel free to reach out via email.  
+
+### Reporting via Email  
+
+If you prefer, you can also report security issues by emailing `express-security@lists.openjsf.org`.  
```

Comment:
@UlisesGascon would you like me to add you as a moderator on this mailing list now? Or I guess I could add the whole @expressjs/security-triage team but since it could generate spam it feels like a subset for the first layer is best. I am still on it as a moderator and luckily this one has not generated much spam.

Reply:
Yes please!

Reply:
I did this yesterday. You should have gotten an email I think. We need to sort out some details though because I think I added you to the email list directly instead of the security triage "committee". I think you have access to this as well right? If not we should get them to add you, then we should sync up to make sure we like how we manage these going forward.